olog/httpx_client.py

```diff
@@ -187,37 +338,107 @@ async def aput_tag(self, name):
+
+        Examples
+        --------
+        >>>cli.put_tag('TEST')
+        {'name': 'TEST1', 'state': 'Active'}
+        '''
         return asyncio.run(self.aput_tag(name))
 
     # Properties
     async def aget_properties(self):
         async with self._session as api:
             res = await api.get('properties')
         res.raise_for_status()
-        return res.json()
+        properties = list()
+        for i in res.json():
+            properties.append(simplify_attr(i))
+        return properties
 
     def get_properties(self):
+        '''
+        Returns
+        -------
+        properties : list
+            A list of property dict
+
+        Examples
+        --------
+        >>>cli.get_properties()
+        '''
         return asyncio.run(self.aget_properties())
 
     async def aget_property(self, name):
         async with self._session as api:
             res = await api.get(f'properties/{name}')
         res.raise_for_status()
-        return res.json()
+        return simplify_attr(res.json())
 
     def get_property(self, name):
+        '''
+        Parameters
+        ----------
+        name : str
+            Name of property.
+
+        Returns
+        -------
+        property : dict
+            A property dict.
+
+        Examples
+        --------
+        >>>cli.get_property('TEST')
+        '''
         return asyncio.run(self.aget_property(name))
 
-    async def aput_properties(self, properties):
+    async def aput_properties(self, named_attributes):
+        properties = list()
+        for name, attributes in named_attributes.items():
+            attr_value = [{'name': ensure_name(name), 'value': value} for name, value in attributes.items()]
+            properties.append(dict({'name': name,
+                                    'owner': self.user,
+                                    'attributes': attr_value}))
         async with self._session as api:
             res = await api.put('properties', json=properties)
         res.raise_for_status()
 
-    def put_properties(self, properties):
-        return asyncio.run(self.aput_properties(properties))
-
-    async def aput_property(self, property):
-        self.check_owner(property)
+    def put_properties(self, named_attributes):
+        '''
+        Parameters
+        ----------
+
+        named_attributes : dict
+            Each element is a pair of {<name>: <attributes>, ...}. <attributes> is dict
+            of <key: value> pair.
+
+        Examples
+        --------
+        >>>cli.put_properties({'TEST0': {'id': None, 'url': None}, 'TEST1':{'id': None, 'url': None}})
+        '''
+        return asyncio.run(self.aput_properties(named_attributes))
+
+    async def aput_property(self, name, attributes):
+        name = ensure_name(name)
+        monitor_list = [name for name, value in attributes.items()
+                        if isinstance(value, int) or isinstance(value, float)]
+        print('monitor_list')
```

Comment:
stray print here and below

Reply:
Thanks. 
btw this is because server would like/user might also want to have a `int` as value of something like `id`. But server always return `'id': '1'`, as a string. So this monitor_list catch who is `int` and convert it before validation.